dds_web/templates/base.html

```diff
@@ -3,69 +3,28 @@
-             </li>
-             <li class="nav-item">
-               <a class="nav-link nav_color" href="#">About</a>
-             </li>
-             {% if g.current_user %}
-             <li class="nav-item">
-                 <a class="nav-link nav_color" href="{{ url_for('user.user_page', loginname=g.current_user) }}">{{ g.current_user }}</a>
-             </li>
-             <li class="nav-item">
-               <a class="nav-link nav_color" href="{{ url_for('user.logout') }}">Logout</a>
-             </li>
-             {% else %}
-             <li class="nav-item">
-               <a class="nav-link nav_color" href="{{ url_for('user.login') }}">Login</a>
-             </li>
-             {% endif %}
-           </ul>
-        </nav>
-
-        <br><br>
-        <div class="container" style="background: white;">
-            {% block body %} Will be replaced by other templates {% endblock body %}
+        {% include 'navbar.html' %}
+        <div class="container pt-2">
+            {% block body %}{% endblock body %}
         </div>
-        
-        <!-- JS and bootstrap loading -->
-        <!--script src="https://kit.fontawesome.com/48759fdc40.js" crossorigin="anonymous"></script-->
-        <script src="https://code.jquery.com/jquery-3.5.1.min.js"
-            integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2"
-            crossorigin="anonymous"></script>
-        <script
-            src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
-            integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
-            crossorigin="anonymous"></script>
-        <script
-            src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
-            integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
-            crossorigin="anonymous"></script>
-        <script src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js">
-        </script>
-        <script src="https://cdn.datatables.net/1.10.24/js/dataTables.bootstrap4.min.js">
-        </script>
-        <!-- Custom JS functions -->
+
+        <!-- Javascript -->
+        <script src="{{ url_for('static', filename='node_modules/jquery/dist/jquery.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='node_modules/bootstrap/dist/js/bootstrap.bundle.min.js') }}"></script>
+        <script src="{{ url_for('static', filename='node_modules/datatables.net/js/jquery.dataTables.min.js') }}"></script>
+        <!-- TODO - bs5 version not yet on npm. When it is, install as an npm package. -->
+        <script src="https://cdn.datatables.net/1.10.24/js/dataTables.bootstrap5.min.js"></script>
+        <!-- We could use FontAwesome through npm, but then we need configure secrets to access the Pro kit which is a hassle -->
+        <script src="https://kit.fontawesome.com/8cd1f5d753.js" crossorigin="anonymous"></script>
```

Comment:
When we added fontawesome to genstat I think we wanted to keep this URL secret?

Reply:
Yeah I did wonder that at first. At the moment it's unlocked so anyone else can use it but we can lock it to a specific domain in the future.

Or if we're worried I can regenerate the ID and we can stick it in a secret. But then that's an additional hassle for anyone wanting to develop the site.